ui/intermittents-view/redux/store.js

```diff
@@ -0,0 +1,7 @@
+import { createStore, applyMiddleware } from "redux";
+import thunk from "redux-thunk";
```

Comment:
@camd what are your thoughts on using redux and redux-thunk? I know you mentioned something about wanting to move the test-view away from redux?

(Perhaps something that there won't be time to change for this PR, but curious for long term)

Reply:
My two cents :) If I had only used React's state there'd be a lot more nested props and many callbacks to update parent components when child components fetch data or need other data to change based on user interactions (date pickers, tree drop down, general UI details).

Reply:
I'm a bit on the fence about redux.  I think it's the case that it may not be the right choice in all situations.  Seems to work well here.  But in the main UI, it SEEMED to be the cause of some slowdown.  I may have just been using it wrong, tbh.  But since the data is so hierarchical, the normal react props are working well.

So, my opinion is that we can take it on a case-by-base basis.  :)  Sounds like I'm sitting on the fence, huh?

I honestly don't know if I'll hit a point where I want to add redux to the main TH ui for some reason.  We've been using events to interact across components thus far and that's working ok.  I think redux would be an (at least partial) alternative approach to that.  But since I'm still 1/2 Angular and we use events there, I'm constrained by it.